ui/lib/core/addon/components/replication-dashboard.hbs

```diff
@@ -71,8 +71,8 @@
       </Hds::Alert>
     </div>
   {{/if}}
-  {{#unless this.isSummaryDashboard}}
-    <ReplicationTableRows @replicationDetails={{this.replicationDetails}} @clusterMode={{this.clusterMode}} />
+  {{#unless @isSummaryDashboard}}
+    <ReplicationTableRows @replicationDetails={{@replicationDetails}} @clusterMode={{this.clusterMode}} />
```

Comment:
it looks like `this.clusterMode` should be an arg, as it's a getter defined in `replication-page.js`